pkg/bootstrap/fuzz_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bootstrap
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"testing"
+
+	fuzz "github.com/AdaLogics/go-fuzz-headers"
+)
+
+func FuzzWriteTo(f *testing.F) {
+	f.Fuzz(func(t *testing.T, data []byte) {
+		ff := fuzz.NewConsumer(data)
```

Comment:
This could be simplified a bit with our wrapper: https://github.com/istio/istio/blob/b524d9836880a88c7aaa91956dfca24c82c29039/pkg/fuzz/README.md#writing-a-test

If you don't like the wrapper for some reason would love feedback, as you have much more experience in this domain than I do.

No problem merging as-is though, its still pretty simple